documentation/source/install/dev_solutions.rst

```diff
@@ -0,0 +1,156 @@
+*******************
+Developer Solutions
+*******************
+
+Minikube
+========
+
+Setting up OpenEBS with Kubernetes using Minikube
+-------------------------------------------------
+
+Minikube helps developers to quickly setup a single-node Kubernetes cluster for their development environement. There are several options available for developers to install Minikube. See `Minikube Setup`_.
+
+       .. _Minikube Setup: https://github.com/kubernetes/minikube
+
+If you are already an experienced Minikube user, you can easily setup OpenEBS on your existing Kubernetes cluster with a few simple kubectl commands. See, :ref:`quick-start`.
+
+This section provides instructions to set up Kubernetes using Minikube directly on Ubuntu 16.06 (without using any VM drivers) and to have OpenEBS running in hyper converged mode. 
+
+Prerequisites
+-------------
+Minimum requirements for using Minikube
+
+* Machine Type - (Minimum 4 vCPUs)
+* RAM - (Minimum 4GB)
+
+Make sure *docker* is installed on your Ubuntu Host. 
+ 
+
+Add iSCSI Support
+-----------------
+
+On your Ubuntu host, install open-iscsi package. OpenEBS uses iSCSI to connect to the block volumes.
+::
+    sudo apt-get update
+    sudo apt-get install open-iscsi
+    sudo service open-iscsi restart
+
+Verify that iSCSI is configured
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Check that initiator name is configured and iSCSI service is running using the following commands.
+::
+   sudo cat /etc/iscsi/initiatorname.iscsi
+   sudo service open-iscsi status
+
+
+Download and setup Minikube and kubectl
+---------------------------------------
+
+On your Ubuntu host, install minikube
+::
+    curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+    chmod +x minikube 
+    sudo mv minikube /usr/local/bin/
+
+On your Ubuntu host, install kubectl
+::
+    curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+    chmod +x kubectl 
+    sudo mv kubectl /usr/local/bin/
+
+On your Ubuntu host, setup directories for storing minkube and kubectl configuration
+::
+    mkdir /home/vagrant/.kube || true
+    touch /home/vagrant/.kube/config
+
+On your Ubuntu host, setup ENV for minikube. Copy the following to ~/.profile
+::
+    export MINIKUBE_WANTUPDATENOTIFICATION=false
+    export MINIKUBE_WANTREPORTERRORPROMPT=false
+    export MINIKUBE_HOME=$HOME
+    export CHANGE_MINIKUBE_NONE_USER=true
+    export KUBECONFIG=$HOME/.kube/config
+
+On your Ubuntu host, start the minikube
```

Comment:
full stop at the end of the sentence.

Reply:
added periods.. for all the above mentioned lines